README.rst

```diff
@@ -562,10 +562,26 @@ ensuring also that ``openwisp_network_topology`` has been removed:
 
     INSTALLED_APPS = [
         # ... other apps ...
-        # 'openwisp_network_topology'  <-- comment out or delete this line
-        'sample_network_topology'
+        'openwisp_utils.admin_theme',
+        # all-auth
+        'django.contrib.sites',
+        'allauth',
+        'allauth.account',
+        'allauth.socialaccount',
+        # controller
+        'openwisp_controller.pki',
+        'openwisp_controller.config',
```

Comment:
openwisp-controller is needed only if `openwisp_network_topology.integrations.device` is used, otherwise it is not required.

Right now, this is not clear from this example. It seems openwisp-controller is required.